pdns/mastercommunicator.cc

```diff
@@ -93,7 +95,7 @@ void CommunicatorClass::queueNotifyDomain(const DNSName &domain, UeberBackend *B
     }
   }
 
-  if (!hasQueuedItem)
+  if (!hasQueuedItem && d_onlyNotify.size())
```

Comment:
Please don't hide the warning if only-notiy is empty, this also applies to also-notify.
If this warning pops up with an empty only-notiy list change the domain type from SLAVE to NATIVE